src/util/uint.rs

```diff
@@ -420,6 +434,13 @@ macro_rules! construct_uint {
 construct_uint!(Uint256, 4);
 construct_uint!(Uint128, 2);
 
+/// Uint error
+#[derive(Debug)]
+pub enum Error {
+    /// Invalid slice length
+    InvalidLength(usize, usize), // (expected, actual)
```

Comment:
nit: should be (actual, expected) as per the above usage.

Reply:
This should probably also be in the actual doc comment (and of course correct).

Reply:
Good catch, fixed in https://github.com/rust-bitcoin/rust-bitcoin/pull/511/commits/46acd7787fbdf135a2c7389f0628f883d7a048da.